proxy/envoy/mixer.go

```diff
@@ -79,7 +79,8 @@ type FilterMixerConfig struct {
 
 func (*FilterMixerConfig) isNetworkFilterConfig() {}
 
-func buildMixerCluster(mesh *proxyconfig.MeshConfig) *Cluster {
+// buildMixerCluster builds an outbound mixer cluster
```

Comment:
I think mixer should take inbound traffic, i.e., proxy calls mixer, not the other way around, right?